sdk/resourcemanager/Azure.ResourceManager/tests/Unit/ManagedServiceIdentityTest.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using Azure.Core;
+using Azure.ResourceManager.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Tests
```

Comment:
Add round trip cases here to serialize with V3 and deserialize with V4 and vice versa